zebra-network/src/constants.rs

```diff
@@ -95,6 +95,15 @@ pub const EWMA_DEFAULT_RTT: Duration = Duration::from_secs(20 + 1);
 /// better peers when we restart the sync.
 pub const EWMA_DECAY_TIME: Duration = Duration::from_secs(200);
 
+/// The maximum time to open a port if it haves a timeout.
+pub const BIND_TIMEOUT: Duration = Duration::from_secs(1);
+
+/// Linux portion of OS error when the port attempting to be opened is already in use.
+pub const PORT_IN_USE_LINUX: &str = "already in use";
+
+/// Windows portion of OS error when the port attempting to be opened is already in use.
+pub const PORT_IN_USE_WINDOWS: &str = "one usage";
```

Comment:
What about macOS?

Reply:
I dont know the macOS strings yet as we were not able to see the panicfor it. We can have `PORT_IN_USE_WINDOWS` and `PORT_IN_USE_LINUX`(instead of `PORT_IN_USE_LINUX`) at least until then. Then see if we need `PORT_IN_USE_MACOS` ?